docs/debugging.md

```diff
@@ -52,7 +52,7 @@ A couple of methods can be used to get debug logging.
 
 This method requires no soldering of board components.
 
-See [Debugging the EC firmware](./ec/doc/debugging.md) for details on setting
+See [Debugging the EC firmware](https://github.com/system76/ec/blob/master/docs/debugging.md) for details on setting
```

Comment:
Since the link makes the line a lot longer, move the rest of the text to the next line.